easybuild/easyblocks/t/tensorflow.py

```diff
@@ -126,7 +127,12 @@ def configure_step(self):
         regex_subs = [(r"(run_shell\(\['bazel')", r"\1, '--output_base=%s'" % tmpdir)]
         apply_regex_substitutions('configure.py', regex_subs)
 
-        run_cmd('./configure', log_all=True, simple=True)
+        cmd = "%(preconfigopts)s ./configure %(configopts)s" % {
+            'preconfigopts': self.cfg['preconfigopts'],
+            'configopts': self.cfg['configopts'],
+        }
```

Comment:
this looks more complex than needed, how about:

```python
cmd = self.cfg['preconfigopts'] +  './configure ' + self.cfg['configopts']
run_cmd(cmd, log_all=True, simple=True)

Reply:
Looks reasonable. I just followed the same approach as in other easyblocks.